polytope/solvers.py

```diff
@@ -88,6 +95,8 @@ def lpsolve(c, G, h, solver=None):
         result = _solve_lp_using_cvxopt(c, G, h, solver=solver)
     elif solver == 'scipy':
         result = _solve_lp_using_scipy(c, G, h)
+    elif solver == 'gurobi':
```

Comment:
Passing the selection `'gurobi'` as a keyword argument as done here is explicit [PEP 8] and seems preferable.